tools/vigiechiro/BilanEnrichiPF.R

```diff
@@ -232,6 +232,6 @@ if(substr(IdC2$`nom du fichier`[1],2,2)!="a")
   
   saveWidget(SummHTMLH,"output-hourly.html")
 #  write.csv2(SummPartH,"output-hourly.tabular",row.names=F)
-  write.table(SummPartH,"output-hourly.tabular",row.names=F,sep="\n")
+  write.table(SummPartH,"output-hourly.tabular",row.names=F,sep="\t")
```

Comment:
Isn't this tested :)

Reply:
The test compare one out of 3 files, it's pretty cheap you're right :D 
I'll had more